webhook_testing_test.go

```diff
@@ -0,0 +1,42 @@
+package braintree
+
+import "testing"
+
+func TestWebhookTestingGatewayRequest(t *testing.T) {
+	t.Parallel()
+
+	var (
+		testingWebhookGateway = testGateway.WebhookTesting()
+		webhookGateway        = testGateway.WebhookNotification()
+	)
+
+	var (
+		method   = "POST"
+		endpoint = "http://1.2.3.4/api/v1/braintree-webhook/"
+		kind     = SubscriptionChargedSuccessfullyWebhook
+		id       = "123"
+	)
```

Comment:
No need to do the `var ( ... = ... )`, `:=` is sufficient and consistent with other test setup.